spec/system/support/better_rails_system_tests.rb

```diff
@@ -0,0 +1,45 @@
+# spec/system/support/better_rails_system_tests.rb
+
+module BetterRailsSystemTests
+  # Use our `Capybara.save_path` to store screenshots with other capybara artifacts
+  # (Rails screenshots path is not configurable https://github.com/rails/rails/blob/49baf092439fc74fc3377b12e3334c3dd9d0752f/actionpack/lib/action_dispatch/system_testing/test_helpers/screenshot_helper.rb#L79)
+  def absolute_image_path
+    Rails.root.join("#{Capybara.save_path}/screenshots/#{image_name}.png")
+  end
+
+  # Make failure screenshots compatible with multi-session setup.
+  # That's where we use Capybara.last_used_session introduced before.
+  def take_screenshot
+    return super unless Capybara.last_used_session
+
+    Capybara.using_session(Capybara.last_used_session) { super }
+  end
+
+  ## Use relative path in screenshot message
+  ##def image_path
+  ##  absolute_image_path.relative_path_from(Rails.root).to_s
+  ##end
+
+end
+
+RSpec.configure do |config|
+  config.include BetterRailsSystemTests, type: :system
+
+  # Make urls in mailers contain the correct server host.
+  # It's required for testing links in emails (e.g., via capybara-email).
+  config.around(:each, type: :system) do |ex|
+    was_host = Rails.application.default_url_options[:host]
+    Rails.application.default_url_options[:host] = Capybara.server_host
+    ex.run
+    Rails.application.default_url_options[:host] = was_host
+  end
+
+  # Make sure this hook runs before others
+  config.prepend_before(:each, type: :system) do
+  # Rails sets host to `127.0.0.1` for every test by default.
+  # That won't work with a remote browser.
+  host! CAPYBARA_APP_HOST
+  # Use JS driver always
+  driven_by Capybara.javascript_driver
```

Comment:
I think these lines inside the `do` / `end` block need to be indented.

Reply:
Removed - see previous comment.